{
  "build": "py311h06a4308_0",
  "build_number": 0,
  "channel": "https://repo.anaconda.com/pkgs/main/linux-64",
  "constrains": [],
  "depends": [
    "jsonschema >=4.18.0",
    "python >=3.11,<3.12.0a0",
    "python-json-logger >=2.0.4",
    "pyyaml >=5.3",
    "referencing",
    "rfc3339-validator",
    "rfc3986-validator >=0.1.1",
    "traitlets >=5.3"
  ],
  "extracted_package_dir": "/home/san/anaconda3/pkgs/jupyter_events-0.10.0-py311h06a4308_0",
  "files": [
    "bin/jupyter-events",
    "lib/python3.11/site-packages/jupyter_events-0.10.0.dist-info/INSTALLER",
    "lib/python3.11/site-packages/jupyter_events-0.10.0.dist-info/METADATA",
    "lib/python3.11/site-packages/jupyter_events-0.10.0.dist-info/RECORD",
    "lib/python3.11/site-packages/jupyter_events-0.10.0.dist-info/REQUESTED",
    "lib/python3.11/site-packages/jupyter_events-0.10.0.dist-info/WHEEL",
    "lib/python3.11/site-packages/jupyter_events-0.10.0.dist-info/direct_url.json",
    "lib/python3.11/site-packages/jupyter_events-0.10.0.dist-info/entry_points.txt",
    "lib/python3.11/site-packages/jupyter_events-0.10.0.dist-info/licenses/LICENSE",
    "lib/python3.11/site-packages/jupyter_events/__init__.py",
    "lib/python3.11/site-packages/jupyter_events/__pycache__/__init__.cpython-311.pyc",
    "lib/python3.11/site-packages/jupyter_events/__pycache__/_version.cpython-311.pyc",
    "lib/python3.11/site-packages/jupyter_events/__pycache__/cli.cpython-311.pyc",
    "lib/python3.11/site-packages/jupyter_events/__pycache__/logger.cpython-311.pyc",
    "lib/python3.11/site-packages/jupyter_events/__pycache__/pytest_plugin.cpython-311.pyc",
    "lib/python3.11/site-packages/jupyter_events/__pycache__/schema.cpython-311.pyc",
    "lib/python3.11/site-packages/jupyter_events/__pycache__/schema_registry.cpython-311.pyc",
    "lib/python3.11/site-packages/jupyter_events/__pycache__/traits.cpython-311.pyc",
    "lib/python3.11/site-packages/jupyter_events/__pycache__/validators.cpython-311.pyc",
    "lib/python3.11/site-packages/jupyter_events/__pycache__/yaml.cpython-311.pyc",
    "lib/python3.11/site-packages/jupyter_events/_version.py",
    "lib/python3.11/site-packages/jupyter_events/cli.py",
    "lib/python3.11/site-packages/jupyter_events/logger.py",
    "lib/python3.11/site-packages/jupyter_events/py.typed",
    "lib/python3.11/site-packages/jupyter_events/pytest_plugin.py",
    "lib/python3.11/site-packages/jupyter_events/schema.py",
    "lib/python3.11/site-packages/jupyter_events/schema_registry.py",
    "lib/python3.11/site-packages/jupyter_events/schemas/event-core-schema.yml",
    "lib/python3.11/site-packages/jupyter_events/schemas/event-metaschema.yml",
    "lib/python3.11/site-packages/jupyter_events/schemas/property-metaschema.yml",
    "lib/python3.11/site-packages/jupyter_events/traits.py",
    "lib/python3.11/site-packages/jupyter_events/validators.py",
    "lib/python3.11/site-packages/jupyter_events/yaml.py"
  ],
  "fn": "jupyter_events-0.10.0-py311h06a4308_0.conda",
  "license": "BSD-3-Clause",
  "link": {
    "source": "/home/san/anaconda3/pkgs/jupyter_events-0.10.0-py311h06a4308_0",
    "type": 1
  },
  "md5": "1c8830f4ce613aa633252dfcbfed858a",
  "name": "jupyter_events",
  "package_tarball_full_path": "/home/san/anaconda3/pkgs/jupyter_events-0.10.0-py311h06a4308_0.conda",
  "paths_data": {
    "paths": [
      {
        "_path": "bin/jupyter-events",
        "file_mode": "text",
        "path_type": "hardlink",
        "prefix_placeholder": "/croot/jupyter_events_1718738097486/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pl",
        "sha256": "276d6713e933d8ad8ccec4f03020d630e0a614fcaab811d6e39d891ca73e96e3",
        "sha256_in_prefix": "03f99148f65fa689731e6b8ac3a522701ba0dfe32b1dfe88ef1d1a505eb8bcc3",
        "size_in_bytes": 471
      },
      {
        "_path": "lib/python3.11/site-packages/jupyter_events-0.10.0.dist-info/INSTALLER",
        "path_type": "hardlink",
        "sha256": "d0edee15f91b406f3f99726e44eb990be6e34fd0345b52b910c568e0eef6a2a8",
        "sha256_in_prefix": "d0edee15f91b406f3f99726e44eb990be6e34fd0345b52b910c568e0eef6a2a8",
        "size_in_bytes": 5
      },
      {
        "_path": "lib/python3.11/site-packages/jupyter_events-0.10.0.dist-info/METADATA",
        "path_type": "hardlink",
        "sha256": "f8ca2d8a79b25b77398e52ef9d3cfae48388744b0ad3f88221dcb330ae9ace33",
        "sha256_in_prefix": "f8ca2d8a79b25b77398e52ef9d3cfae48388744b0ad3f88221dcb330ae9ace33",
        "size_in_bytes": 5714
      },
      {
        "_path": "lib/python3.11/site-packages/jupyter_events-0.10.0.dist-info/RECORD",
        "path_type": "hardlink",
        "sha256": "1bdbd1a1cdfcde1396e975bedfb6adcba2996c80af030e9d10f7da493bf9e370",
        "sha256_in_prefix": "1bdbd1a1cdfcde1396e975bedfb6adcba2996c80af030e9d10f7da493bf9e370",
        "size_in_bytes": 2591
      },
      {
        "_path": "lib/python3.11/site-packages/jupyter_events-0.10.0.dist-info/REQUESTED",
        "path_type": "hardlink",
        "sha256": "e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855",
        "sha256_in_prefix": "e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855",
        "size_in_bytes": 0
      },
      {
        "_path": "lib/python3.11/site-packages/jupyter_events-0.10.0.dist-info/WHEEL",
        "path_type": "hardlink",
        "sha256": "4c93e729db6b4ae7bbc59fc0546929f585dcbc3ae86ec8c176cd5dbb7371e9d7",
        "sha256_in_prefix": "4c93e729db6b4ae7bbc59fc0546929f585dcbc3ae86ec8c176cd5dbb7371e9d7",
        "size_in_bytes": 87
      },
      {
        "_path": "lib/python3.11/site-packages/jupyter_events-0.10.0.dist-info/direct_url.json",
        "path_type": "hardlink",
        "sha256": "70b974e0172f691cfe3988e06a790601e2fb004c75b82477fe79dcd374488d66",
        "sha256_in_prefix": "70b974e0172f691cfe3988e06a790601e2fb004c75b82477fe79dcd374488d66",
        "size_in_bytes": 74
      },
      {
        "_path": "lib/python3.11/site-packages/jupyter_events-0.10.0.dist-info/entry_points.txt",
        "path_type": "hardlink",
        "sha256": "da6e32437af24d8a74f2e98251e5c1a90d4b2a2c37b696c93ecb93ccf9f288de",
        "sha256_in_prefix": "da6e32437af24d8a74f2e98251e5c1a90d4b2a2c37b696c93ecb93ccf9f288de",
        "size_in_bytes": 59
      },
      {
        "_path": "lib/python3.11/site-packages/jupyter_events-0.10.0.dist-info/licenses/LICENSE",
        "path_type": "hardlink",
        "sha256": "80405b5460d62fb7032d62e4720fa199a5be01864c900f1b9de79bc811efd836",
        "sha256_in_prefix": "80405b5460d62fb7032d62e4720fa199a5be01864c900f1b9de79bc811efd836",
        "size_in_bytes": 1534
      },
      {
        "_path": "lib/python3.11/site-packages/jupyter_events/__init__.py",
        "path_type": "hardlink",
        "sha256": "ef1fb7618c7b8626bd723fdf7f1d52e69c4f62d108fdd14ac1bd19e43d9e2b0f",
        "sha256_in_prefix": "ef1fb7618c7b8626bd723fdf7f1d52e69c4f62d108fdd14ac1bd19e43d9e2b0f",
        "size_in_bytes": 222
      },
      {
        "_path": "lib/python3.11/site-packages/jupyter_events/__pycache__/__init__.cpython-311.pyc",
        "path_type": "hardlink",
        "sha256": "a6df6f4f77909f8bdb54f5bf8a2d738ff919b1f4b492a18eb1f343e9ae0eadc4",
        "sha256_in_prefix": "a6df6f4f77909f8bdb54f5bf8a2d738ff919b1f4b492a18eb1f343e9ae0eadc4",
        "size_in_bytes": 426
      },
      {
        "_path": "lib/python3.11/site-packages/jupyter_events/__pycache__/_version.cpython-311.pyc",
        "path_type": "hardlink",
        "sha256": "fbda0ef581571d6f3be2e41460e611d3f11c6fcd95d72e83299da46dd2471fd7",
        "sha256_in_prefix": "fbda0ef581571d6f3be2e41460e611d3f11c6fcd95d72e83299da46dd2471fd7",
        "size_in_bytes": 1155
      },
      {
        "_path": "lib/python3.11/site-packages/jupyter_events/__pycache__/cli.cpython-311.pyc",
        "path_type": "hardlink",
        "sha256": "1fef716f49d442b66c79bf6133c5e3753fc8479089a122c8279b92d185648713",
        "sha256_in_prefix": "1fef716f49d442b66c79bf6133c5e3753fc8479089a122c8279b92d185648713",
        "size_in_bytes": 5401
      },
      {
        "_path": "lib/python3.11/site-packages/jupyter_events/__pycache__/logger.cpython-311.pyc",
        "path_type": "hardlink",
        "sha256": "8a72b9a44a482bca96ec232176f90d0cb86ec9e85fd5d6760ed497cb4b593213",
        "sha256_in_prefix": "8a72b9a44a482bca96ec232176f90d0cb86ec9e85fd5d6760ed497cb4b593213",
        "size_in_bytes": 18467
      },
      {
        "_path": "lib/python3.11/site-packages/jupyter_events/__pycache__/pytest_plugin.cpython-311.pyc",
        "path_type": "hardlink",
        "sha256": "c2b24aed3fa5e5d23194f59f497c851df0a3aafc16c16237b9763a01f2c3ce27",
        "sha256_in_prefix": "c2b24aed3fa5e5d23194f59f497c851df0a3aafc16c16237b9763a01f2c3ce27",
        "size_in_bytes": 3416
      },
      {
        "_path": "lib/python3.11/site-packages/jupyter_events/__pycache__/schema.cpython-311.pyc",
        "path_type": "hardlink",
        "sha256": "2c2a6584c77f7bc0f30e3e6bf0f9d5abfef0351b9ac3c292ad80b47e2fc73ac1",
        "sha256_in_prefix": "2c2a6584c77f7bc0f30e3e6bf0f9d5abfef0351b9ac3c292ad80b47e2fc73ac1",
        "size_in_bytes": 7604
      },
      {
        "_path": "lib/python3.11/site-packages/jupyter_events/__pycache__/schema_registry.cpython-311.pyc",
        "path_type": "hardlink",
        "sha256": "5631a3441780ade33db72ca7ed28fcf1a071961758f8313e9fe8feeea9775741",
        "sha256_in_prefix": "5631a3441780ade33db72ca7ed28fcf1a071961758f8313e9fe8feeea9775741",
        "size_in_bytes": 4979
      },
      {
        "_path": "lib/python3.11/site-packages/jupyter_events/__pycache__/traits.cpython-311.pyc",
        "path_type": "hardlink",
        "sha256": "312c367e0ed636c5de89034a44e9a5b6934e083ec0d3aef593499330a54dc80d",
        "sha256_in_prefix": "312c367e0ed636c5de89034a44e9a5b6934e083ec0d3aef593499330a54dc80d",
        "size_in_bytes": 2596
      },
      {
        "_path": "lib/python3.11/site-packages/jupyter_events/__pycache__/validators.cpython-311.pyc",
        "path_type": "hardlink",
        "sha256": "a3ba3f974a9c9f982b859cf03a0f72e0ef115c6cb2a878b08e7cf343ebba0722",
        "sha256_in_prefix": "a3ba3f974a9c9f982b859cf03a0f72e0ef115c6cb2a878b08e7cf343ebba0722",
        "size_in_bytes": 3307
      },
      {
        "_path": "lib/python3.11/site-packages/jupyter_events/__pycache__/yaml.cpython-311.pyc",
        "path_type": "hardlink",
        "sha256": "ee44da3a5fd67545e8d1a5fd54de64889a659d83ee0f6c4fe187db921352fbf2",
        "sha256_in_prefix": "ee44da3a5fd67545e8d1a5fd54de64889a659d83ee0f6c4fe187db921352fbf2",
        "size_in_bytes": 2074
      },
      {
        "_path": "lib/python3.11/site-packages/jupyter_events/_version.py",
        "path_type": "hardlink",
        "sha256": "1379afb135572376d3f7da8960f7113b3bf7e26868b51613767ae0462e8cc755",
        "sha256_in_prefix": "1379afb135572376d3f7da8960f7113b3bf7e26868b51613767ae0462e8cc755",
        "size_in_bytes": 648
      },
      {
        "_path": "lib/python3.11/site-packages/jupyter_events/cli.py",
        "path_type": "hardlink",
        "sha256": "fc226693bd54009e135f5c3b5a247ffe150a205064cbc8f62b67aeab4658d177",
        "sha256_in_prefix": "fc226693bd54009e135f5c3b5a247ffe150a205064cbc8f62b67aeab4658d177",
        "size_in_bytes": 3087
      },
      {
        "_path": "lib/python3.11/site-packages/jupyter_events/logger.py",
        "path_type": "hardlink",
        "sha256": "3092bce56b33bdf76cb05990d4f35b1126d63e02ae51524a2e9c38906006777f",
        "sha256_in_prefix": "3092bce56b33bdf76cb05990d4f35b1126d63e02ae51524a2e9c38906006777f",
        "size_in_bytes": 15842
      },
      {
        "_path": "lib/python3.11/site-packages/jupyter_events/py.typed",
        "path_type": "hardlink",
        "sha256": "e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855",
        "sha256_in_prefix": "e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855",
        "size_in_bytes": 0
      },
      {
        "_path": "lib/python3.11/site-packages/jupyter_events/pytest_plugin.py",
        "path_type": "hardlink",
        "sha256": "1b4182cfb43fdf21ed02df73d00a2a64d57e6d06efe79defbe3d8c303cf2ede2",
        "sha256_in_prefix": "1b4182cfb43fdf21ed02df73d00a2a64d57e6d06efe79defbe3d8c303cf2ede2",
        "size_in_bytes": 1778
      },
      {
        "_path": "lib/python3.11/site-packages/jupyter_events/schema.py",
        "path_type": "hardlink",
        "sha256": "00bebc037267c3d40830ccde331b310d9833b906f2201e47b4cd13e9bb43e019",
        "sha256_in_prefix": "00bebc037267c3d40830ccde331b310d9833b906f2201e47b4cd13e9bb43e019",
        "size_in_bytes": 5390
      },
      {
        "_path": "lib/python3.11/site-packages/jupyter_events/schema_registry.py",
        "path_type": "hardlink",
        "sha256": "0a78b0a27490a3731cd2d44b2fe89576c9ada1a0788dfb8a9a67492bc6e3f78d",
        "sha256_in_prefix": "0a78b0a27490a3731cd2d44b2fe89576c9ada1a0788dfb8a9a67492bc6e3f78d",
        "size_in_bytes": 2762
      },
      {
        "_path": "lib/python3.11/site-packages/jupyter_events/schemas/event-core-schema.yml",
        "path_type": "hardlink",
        "sha256": "c4fc7e1b9d162386463dde84fdb6564293503c71ac306b29d808b0cbcc6baccb",
        "sha256_in_prefix": "c4fc7e1b9d162386463dde84fdb6564293503c71ac306b29d808b0cbcc6baccb",
        "size_in_bytes": 580
      },
      {
        "_path": "lib/python3.11/site-packages/jupyter_events/schemas/event-metaschema.yml",
        "path_type": "hardlink",
        "sha256": "721d42ebc8de49053263a4ba000a51f701670236f1dbe79cf9b380b2e8ac50eb",
        "sha256_in_prefix": "721d42ebc8de49053263a4ba000a51f701670236f1dbe79cf9b380b2e8ac50eb",
        "size_in_bytes": 626
      },
      {
        "_path": "lib/python3.11/site-packages/jupyter_events/schemas/property-metaschema.yml",
        "path_type": "hardlink",
        "sha256": "9a2ab66d1473a3d9bf7a154ad6f6316f0c78e91847e2e0e5212fa5a72bc686e3",
        "sha256_in_prefix": "9a2ab66d1473a3d9bf7a154ad6f6316f0c78e91847e2e0e5212fa5a72bc686e3",
        "size_in_bytes": 624
      },
      {
        "_path": "lib/python3.11/site-packages/jupyter_events/traits.py",
        "path_type": "hardlink",
        "sha256": "f78753814a2a4d36a8fc73cfbc701853069e2aaaf8154115dbb8d5ba7120a603",
        "sha256_in_prefix": "f78753814a2a4d36a8fc73cfbc701853069e2aaaf8154115dbb8d5ba7120a603",
        "size_in_bytes": 1818
      },
      {
        "_path": "lib/python3.11/site-packages/jupyter_events/validators.py",
        "path_type": "hardlink",
        "sha256": "94f27a8c350b51e9839c617b8e778de6ca3629391e3c8c5e08f3b7408d1403bb",
        "sha256_in_prefix": "94f27a8c350b51e9839c617b8e778de6ca3629391e3c8c5e08f3b7408d1403bb",
        "size_in_bytes": 2377
      },
      {
        "_path": "lib/python3.11/site-packages/jupyter_events/yaml.py",
        "path_type": "hardlink",
        "sha256": "13fb01b745636e1a45caed4b302806db3d59b57eba0b26a3f989130a37534469",
        "sha256_in_prefix": "13fb01b745636e1a45caed4b302806db3d59b57eba0b26a3f989130a37534469",
        "size_in_bytes": 1031
      }
    ],
    "paths_version": 1
  },
  "requested_spec": "None",
  "sha256": "620c2addf3f101eb6c63d07309e6966b1e6589ecb5ad16413277ee529dafa5ec",
  "size": 43680,
  "subdir": "linux-64",
  "timestamp": 1718738122000,
  "url": "https://repo.anaconda.com/pkgs/main/linux-64/jupyter_events-0.10.0-py311h06a4308_0.conda",
  "version": "0.10.0"
}